pkg/proto/datadog/api/README.md

```diff
@@ -11,7 +11,7 @@ files we will need three different grpc-related packages:
 
 From the repository root run the following:
 ```
-inv install-tools
+inv setup
```

Comment:
note: `install-tools` doesnt trigger the installation of the tools defined in the `tools.go` file. for that you need to run `inv setup`